Makefile

```diff
@@ -145,7 +145,7 @@ test: generate fmt vet gotestsum ## Run tests.
 
 .PHONY: test-integration
 test-integration: manifests generate fmt vet envtest ginkgo mpi-operator-crd ## Run tests.
-	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) --arch=amd64 use $(ENVTEST_K8S_VERSION) -p path)" \
+	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" \
```

Comment:
Since k8s v1.24, envtest has started supporting arm64 architecture.

ref: https://github.com/kubernetes-sigs/kubebuilder/issues/2664

Reply:
Could be a separate PR for better tracking